src/index.ts

```diff
@@ -1,3 +1,49 @@
-import { Headline, H1, H2, H3, H4, H5, Body1, Body2, Body3, Label1, Label2, Label3 } from "./components/Typography";
+import {
+  Headline,
+  H1,
+  H2,
+  H3,
+  H4,
+  H5,
+  Body1,
+  Body2,
+  Body3,
+  Label1,
+  Label2,
+  Label3,
```

Comment:
Maybe we should rename to `BodyLg` or `BodyLarge`, `BodyMd` etc... ? Will be easier match components with the Design System. The suffix `1, 2` says nothing. For heading tags `1,2..` works for me because we have HTML tags like `h1, h2, h3...`